### PR TITLE
Fix the circle adapter to prevent writing undefined

### PR DESCRIPTION
**Pull request by wayfarer3130**
The circle adapter was writing undefined for the area and diameter.

### CODE REVIEW

**Review by pieper — APPROVED** [review]
I have no problem with this.  Does anyone from the cornerstone group wish to review?  @sedghi @JamesAPetts

File: src/adapters/Cornerstone/CircleRoi.js [hunk 1]
```diff
@@ -29,7 +28,9 @@ class CircleRoi {
             toolType: CircleRoi.toolType,
             active: false,
             cachedStats: {
-                area: NUMGroup.MeasuredValueSequence.NumericValue
+                area: NUMGroup.MeasuredValueSequence.NumericValue,
+                radius: 5,
```
Comment (wayfarer3130) [inline src/adapters/Cornerstone/CircleRoi.js:32]: Needed dummy values here - eventually should use the right set of values read from the response.

File: src/adapters/Cornerstone/CircleRoi.js [hunk 2]
```diff
@@ -67,8 +68,9 @@ class CircleRoi {
     static getTID300RepresentationArguments(tool) {
         const { cachedStats, handles, finding, findingSites } = tool;
         const { start: center, end } = handles;
-        const { area } = cachedStats;
+        const { area, radius } = cachedStats;
```
Comment (wayfarer3130) [inline src/adapters/Cornerstone/CircleRoi.js:71]: Read the values as specified.

**Comment by sedghi** [comment]
> @sedghi - could you respond to Steve please about this issue? It is just an adapter that works for cornerstone 2d circle roi, as the previous one had some bugs in it.

I would ask @JamesAPetts since he knows better this space, my only comment is that why we are hard coding radius: 5

File: src/adapters/Cornerstone/CircleRoi.js (hunk 1 above)
Comment (wayfarer3130) [reply src/adapters/Cornerstone/CircleRoi.js]: Changed to be 0 @sedghi  - added a comment to the code explaining why it has the value 0 as it is jsut a dummy.

**Comment by JamesAPetts** [comment]
LGTM. I agree with setting the values 0 if we don't know them, its the only honest thing to do.

**Pull request merged**